core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponentSpec.scala

```diff
@@ -1141,6 +1141,23 @@ class EntityComponentSpec extends TestDriverComponentWithFlatSpecAndMatchers wit
     }
   }
 
+  it should "return false if the attribute name does not exist" in withMinimalTestDatabase { _ =>
+    withWorkspaceContext(testData.workspace) { context =>
+      val exists = runAndWait(entityQuery.doesAttributeNameAlreadyExist(context, "Pair", AttributeName.withDefaultNS("case2"))).get
+      assert(!exists)
+    }
+  }
```

Comment:
nit: these tests will result in output of:
```
[info] EntityComponentSpec:
[info] EntityComponent
[info] - should return false if the attribute name does not exist
[info] - should change the attribute name
```
which isn't too helpful for anyone reading the test results. Either add more info into the test name itself, like:
```
it should "return false from doesAttributeNameAlreadyExist if the attribute name does not exist" in withMinimalTestDatabase { _ =>
```
or get all Scalatest on it and do something like:
```
behavior of "doesAttributeNameAlreadyExist()"

it should "return false if the attribute name does not exist" in withMinimalTestDatabase { _ =>
```
but note that if you set `behavior of ` it will affect ALL tests underneath this one, so that's a pain in the butt